src/utils/expression-runner.ts

```diff
@@ -75,7 +80,7 @@ export function evaluateExpression(
     _,
   };
 
-  expression = linkReferencedFieldValues(fields, fieldValues, parts);
+  expression = expressionFormatter(linkReferencedFieldValues(fields, fieldValues, parts), fields, fieldValues);
```

Comment:
@arodidev based on our conversion a few days ago. Does the splitting remain the same? and the only difference is the expression being resolved regardless if you have spaces or not?

Reply:
@pirupius pius Yes, that is correct. The formatting of the expression remains unaffected, but rather a `formField` namespace is prefixed to the field id.